services/pane.test.js

```diff
@@ -156,54 +157,107 @@ describe(dirname, function () {
       });
     });
 
-    describe('openValidationErrors', function () {
-      var fn = lib[this.title],
-        sandbox;
+    describe('openNewPage', function () {
+      var mock = {
+          locals: {edit: true}
+        },
+        noFilter = function () {},
+        createPage, el, env, getState, result, sandbox, spy, stub, template;
 
       before(function () {
-        var template = document.createElement('template'),
-          headEl = document.createElement('div'),
-          paneInnerEl = document.createElement('div'),
-          toolbar = document.createElement('div');
-
-        // add template
-        template.classList.add('kiln-pane-template');
-        headEl.classList.add('pane-header');
-        paneInnerEl.classList.add('pane-inner');
-        template.content.appendChild(headEl);
-        template.content.appendChild(paneInnerEl);
-        document.body.appendChild(template);
-
-        // add toolbar
-        toolbar.classList.add('kiln-toolbar');
-        document.body.appendChild(toolbar);
+        env = new nunjucks.Environment();
+        // satisfy request for nunjucks filter in template
+        env.addFilter('includeFile', noFilter); // TODO – include nunjucks filters
```

Comment:
Satisfies the test's hunger for nunjucks filters. We currently only need the dom elements so they aren't necessary, but it's something we need to figure out.